nengo_viz/static/viz_sim_control.js

```diff
@@ -9,17 +9,17 @@
 VIZ.SimControl = function(div, args) {
     var self = this;
 
-    div.classList.add('sim_control');    
+    div.classList.add('sim_control');
     this.div = div;
     div.style.position = 'fixed';
-    //TODO: Is this the method we want to use for keeping simcontrol on top 
+    //TODO: Is this the method we want to use for keeping simcontrol on top
     //or do we want to dynamically raise it's zindex when other elements have theres increased
-    div.style.zIndex = 99999999; 
+    div.style.zIndex = 1040;
```

Comment:
Bootstrap's modal has a hard-coded z-index of 1050. While we can change that for the div itself, it wasn't obvious to me how to change it for the shaded background, so unless we change this z-index, the sim_control will show up under the modal, but over the background shading. It could be possible to dig a bit deeper and figure out how to set the background z-index higher; do we expect to get z-indexes over 1050 anywhere?